sdk/servicebus/azure-messaging-servicebus/docs/log4j2.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="INFO">
+  <Appenders>
+    <Console name="Console" target="SYSTEM_OUT">
+      <PatternLayout pattern="%d{HH:mm:ss} [%t] %level %logger{1.} - %msg%n"/>
+    </Console>
+    <RollingFile name="RollingFile" fileName="logs/debug.log"
+                 filePattern="logs/$${date:yyyy-MM}/app-%d{MM-dd-yyyy}-%i.log.gz">
+      <PatternLayout>
+        <Pattern>%d{HH:mm:ss} [%t] %level %logger{1.} - %msg%n"</Pattern>
+      </PatternLayout>
+      <Policies>
+        <TimeBasedTriggeringPolicy/>
+        <SizeBasedTriggeringPolicy size="25 MB"/>
+      </Policies>
+    </RollingFile>
+  </Appenders>
+
+  <!-- Additivity is not set here so we inherit the existing Appenders from the Root element. -->
+  <Loggers>
+    <!-- Log only warnings from the checkpoint store. -->
+    <Logger name="com.azure.messaging.eventhubs.checkpointstore.blob" level="WARN">
+    </Logger>
+
+    <!-- PartitionBasedLoadBalancer: Log warnings. The usual output is load balancing output which is noisy. -->
+    <Logger name="com.azure.messaging.eventhubs.PartitionBasedLoadBalancer" level="WARN">
+    </Logger>
+
+    <!-- OPTIONAL SUPPRESSION. If we know it is not a link credit or propagation issue, we can suppress this. -->
+    <!-- Log only the 'On next' messages so we know messages are being propagated downstream. -->
+    <Logger name="com.azure.messaging.eventhubs.PartitionPumpManager" level="DEBUG">
```

Comment:
I think we can remove these... are there other instances where there is too much logging in ServiceBusProcessorClient?

Reply:
In a few cases the verbose logging is helpful. We might need to revisit this after the logging reduction work we have in the pipeline